src/web/DomElement.C

```diff
@@ -734,7 +734,7 @@ void DomElement::setJavaScriptEvent(EscapeOStream& out,
   // events on the dom root container are events received by the whole
   // document when no element has focus
 
-  int fid = nextId_++;
+  unsigned fid = nextId_++;
```

Comment:
So all of them use `.fetch_add(1, std::memory_order_relaxed)` except for this one...

Reply:
That's actually an oversight on my end...